src/NuGet.Clients/NuGet.Tools/GlobalSuppressions.cs

```diff
@@ -30,3 +30,4 @@
 [assembly: SuppressMessage("Design", "CA1001:Types that own disposable fields should be disposable", Justification = "IVsPackage.Close() method invokes overridden NuGetPackage.Dispose() method upon closing the IDE.", Scope = "type", Target = "~T:NuGetVSExtension.NuGetPackage")]
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGetVSExtension.NuGetPackage.Search(Microsoft.VisualStudio.Shell.Interop.IVsWindowFrame,System.String)")]
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGetVSExtension.NuGetPackage.ShowUpdatePackages(Microsoft.VisualStudio.Shell.Interop.IVsWindowFrame,NuGet.PackageManagement.UI.ShowUpdatePackageOptions)")]
+[assembly: SuppressMessage("Style", "IDE0051:Remove unused private members", Justification = "<Pending>", Scope = "member", Target = "~P:NuGetVSExtension.NuGetPackage.SourceControlTracker~System.Lazy{NuGet.VisualStudio.IVsSourceControlTracker}")]
```

Comment:
I'm really not a fan of GlobalSuppressions.cs.  Can we use inline suppressions instead?

Although I'm also curious why this method isn't deleted.